modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -14,8 +20,7 @@ You can configure the `FlowCollector` to filter eBPF flows using a global rule t
 . Configure the `FlowCollector` custom resource, similar to the following sample configurations:
 +
 
-[%collapsible]
-.Filter Kubernetes service traffic to a specific Pod IP endpoint
+.Example YAML to sample all North-South traffic, and 1:50 East-West traffic
 ====
```

Comment:
Any place you remove the `[%collapsible]` line, you want to get rid of the `====` lines that were there to define the collapsible block start and end